tests/subgraph/mapping-v3-core/helpers.ts

```diff
@@ -272,6 +275,16 @@ export function mockMinterUpdatedCallsNoPreconfiguredProjects(
     "getNumProjectsWithMinters",
     "getNumProjectsWithMinters():(uint256)"
   ).returns([ethereum.Value.fromI32(0)]);
+
+  // should handle unsupported minterFilterType function, since some legacy
+  // minter filters (including flagship MinterFilterV1) don't implement the
+  // function
+  // @dev shared minter types not needed, since e2e testing is used for those minters
+  createMockedFunction(
+    TEST_MINTER_FILTER_ADDRESS,
+    "minterFilterType",
+    "minterFilterType():(string)"
+  ).reverts();
```

Comment:
Although the current MinterFilterV1 in the contracts repo does implement a minterFilterType function, the originally deployed MinterFilterV1 used by our flagship V3 core did not implement the function. Therefore, we should handle the revert case gracefully.